src/sg/syzkaller1.c

```diff
@@ -34,7 +34,7 @@
 #include <linux/if_ether.h>
 #include <linux/if_tun.h>
 #include <linux/ip.h>
-#include <linux/kvm.h>
+//#include <linux/kvm.h>
```

Comment:
Yup, this is unnecessary. It's probably syzkaller-generated boilerplate. Could you please update your commit to remove the line entirely instead of commenting it out?